src/AppProviders.tsx

```diff
@@ -11,6 +11,7 @@ import { WagmiProvider } from 'wagmi'
 import { ScrollToTop } from './Routes/ScrollToTop'
 
 import { ChatwootWidget } from '@/components/ChatWoot'
+import { ActionCenterProvider } from '@/components/Layout/Header/ActionCenter/ActionCenterContext'
```

Comment:
Now lives as a context so we're not forced to have everything action center as children of `<ActionCenter />` to use `onDrawerOpen` and `isDrawerOpen`